### ISSUE TITLE
Manual position changes when changing folder name

### ISSUE THREAD

**Issue by jmavra** [getgrav/grav-plugin-admin#1081]
When I change the folder name of a folder, and it is in position 04 (out of 05) (with positioning enabled). When I click save, it moves the folder to the end always (so it changes its number to 05 and the one at 05 becomes 04.

This doesn't seem very user friendly, as I didn't notice until I refreshed the main page.

Thanks,
John

**Comment by flaviocopes** [comment]
Mmm, cannot replicate. It's an empty folder, without markdown file?

**Comment by jmavra** [comment]
Yes, I did create new folder, then I added 2 pages to that folder. The folder was named Service and I renamed it to Services, then every time I renamed it back and forth - Services moves in the position table. If you cannot reproduce let me know and ill send you my user folder.

**Comment by D14RAP** [comment]
this occurs when anything on the page is changed, even uploading media

**Comment by rhukster** [comment]
I definitely can see that if you change order when you also rename the folder is causing order change to be ignored.  however, it's not fatal.  You can change order and save again.  It's just that changing order AND changing folder are 2 quite bit changes, and currently they are mutually exclusive so bad things to don't happen.  This is something that is probably fixable though.  I'll look into it after the next release.

**Comment by D14RAP** [comment]
Great to hear, i know its not urgent but can become very annoying especially in my grave workload using it to make a one page portfolio. Thanks for the look!
